src/test/java/com/example/shift/left/quality/AppTest.java

```diff
@@ -0,0 +1,15 @@
+package com.example.shift.left.quality;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest {
+	@Test
+	public void testGreetWithWorld() {
+		assertEquals("Hello World!", App.greet("World!"));
+	}
```

Comment:
How do you ensure, the behaviour of the  method "greet" with parameters of various different types eg .. 

greet("")
greet(null)
greet(" ")

etc ....